### PR TITLE
Set Default warehouse Supplier - U  in  Purchase  Order.

### PR DESCRIPTION
**Pull request by PITPL-Gayatri-Gajjjar**

### CODE REVIEW

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]
need only single line change in warehouse field. this type of changes are dangerous for migration.

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]
remove indention change and push only required changes. this change harmful.

File: erpnext/buying/doctype/purchase_order/purchase_order.json [hunk 1]
```diff
@@ -1,1298 +1,1299 @@
...
-     }
-    ],
-    "search_fields": "status, transaction_date, supplier, grand_total",
-    "show_name_in_global_search": 1,
-    "sort_field": "modified",
-    "sort_order": "DESC",
-    "track_changes": 1
-   }
\ No newline at end of file
+ "actions": [],
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/buying/doctype/purchase_order/purchase_order.json]: don't push indention change.

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]

File: erpnext/buying/doctype/purchase_order/purchase_order.json (hunk 1 above)
Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/buying/doctype/purchase_order/purchase_order.json]: Done

Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/buying/doctype/purchase_order/purchase_order.json]: Done

**Pull request closed without merging**